app/test_main.py

```diff
@@ -1,3 +1,17 @@
 from app.main import get_human_age
 
-# write your code here
+
+def test_animal_age_is_less_or_equal_14() -> None:
+    assert get_human_age(14, 14) == [0, 0]
+
+
+def test_animal_age_is_less_or_equal_23() -> None:
+    assert get_human_age(23, 23) == [1, 1]
+
+
+def test_animal_is_less_or_equal_27() -> None:
+    assert get_human_age(27, 27) == [2, 2]
+
+
+def test_animal_is_bigger_equal_27() -> None:
+    assert get_human_age(28, 28) == [3, 2]
```

Comment:
use @pytest.mark.parametrize instead of creating multiple tests

Make sure you've tests checks all edged situations such as:

if output should change with some integer value you should check that it is not changed with previous value;
if function resive data out of normal range, such as negative numbers, zero, or realy large numbers;
if the function receives an incorrect type of data it raises the correct exception.